input/docs/extending/modules/best-practices.md

```diff
@@ -162,35 +162,41 @@ Until early 2019, the recommendation was to reference the Cake Contrib icon host
 This recommendation changed once again in the fall of 2019 when NuGet started supporting embedded icons.
 :::
 
+## NuGet compatible tags
+
+**_§4.1_** **Do** add `cake-module` to the NuGet-Tags.
```

Comment:
Continue with 3.x (as it is still part of the package metadata section)

Reply:
👍